apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-steps/plan-step/index.tsx

```diff
@@ -20,20 +20,18 @@ const PlanStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep, onN
 	return (
 		<LaunchStep className="nux-launch-plan-step">
 			<div className="nux-launch-step__header">
-				<div className="nux-launch-step__heading">
-					<h1 className="nux-launch-step__title">{ __( 'Choose a plan', 'full-site-editing' ) }</h1>
-					<p className="nux-launch-step__subtitle">
+				<div>
+					<Title>{ __( 'Select a plan', 'full-site-editing' ) }</Title>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Select a plan' )` *ES Score: 6*